lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'order'
```

Comment:
Do you need this? Even tough the Order class isn't instantiated from this file?

Reply:
You are right this isnt needed also. Thanks!